views/index.hbs

```diff
@@ -1,4 +1,4 @@
-﻿<h1 class="f1 f-6-ns orange tc"><a class="orange no-underline" href="/">Hacktoberfest Checker</a></h1>
```

Comment:
This causes the main logo to be left aligned so don't think we want to remove this one?